lunchable/plugins/splitlunch/lunchmoney_splitwise.py

```diff
@@ -1080,12 +1045,17 @@ def filter_relevant_splitwise_expenses(
 
         1) It filters out deleted expenses
 
-        2) It filters out `self-paid` expenses. A `self-paid` expense is an expense in Splitwise
-        where you originated the payment. This is excluded because it is assumed that these
-        transactions will have already been imported via a different account.
+        2) It filters out expenses with a financial impact of 0, implying that the user was not
+        involved in the expense.
+
+        3) If the --allow-self-paid flag is not provided, it filters out `self-paid` expenses. A
+        `self-paid` expense is an expense in Splitwise where you originated the payment. This is
+        excluded because it is assumed that these transactions will have already been imported via a
+        different account.
 
-        3) It filters out payments. Payments are excluded because it is assumed that these
-        transactions will have already been imported via a different account.
+        4) If the --allow-payments flag is not provided, it filters out payments. Payments are
+        excluded because it is assumed that these transactions will have already been imported via a
+        different account.
```

Comment:
The content in these docstrings is great - is there any of this that's worth surfacing in the public facing plugin docs @ [docs/source/splitlunch.rst](https://github.com/juftin/lunchable/blob/main/docs/source/splitlunch.rst). (I know, it's an `RST` and not `MD` 😞)?

Reply:
Sure, seems reasonable. Added some stuff.